geonode/maps/templates/leaflet_maps/map_edit.html

```diff
@@ -386,6 +375,16 @@ <h4 class="modal-title">Map Updated</h4>
                 new_background_layer.addTo(map);
                 new_background_layer.setZIndex(0);
                 active_background_layer = new_background_layer;
+                // clear all visibility state
+                for (i = 0; i < all_layers_list.length; i++) {
```

Comment:
Additional code to toggle all the background layer state to visibility = false, before setting the used one into true.

Reply:
perhaps setting all the visibility state to false not a good idea.
i suspect this part is the cause of edit map does not show up on a "fresh" created map.
we must be careful with the edit maps because there are some use cases here.
i'm not sure about this, so i still need time to check on `qgis-server-2.7-testing-1.4` tag.

Reply:
Maybe we will just fix it if there is still an issue. For now, it is needed so the edit map can work properly.